tests/tests3.cpp

```diff
@@ -121,6 +121,19 @@ TEST_CASE("Wide string") {
     }
 }
 #endif  //  SQLITE_ORM_OMITS_CODECVT
+
+TEST_CASE("sqlite_schema") {
```

Comment:
idea for reducing amount of tests inside `tests*.cpp` files: we can move `TEST_CASE("sqlite_schema")` and `TEST_CASE("dbstat")` into a dedicated file called how you call such kind of tables